roles/veyepar-encoder/tasks/main.yml

```diff
@@ -3,36 +3,65 @@
   apt:
     name: "{{ item }}"
   with_items:
+  - python-psycopg2
```

Comment:
>  I'll assume you mean the local one that is setup elseware here with requirements.txt, not one on another box.

That's the only one that's relevant, surely?

> I waffle between using apt vs pip to install things, and --site-packages, and often I use both.

Right, but in this case, surely we don't have to do that? It's not like we're installing all the other dependencies system wide, so it's not going to work...